README.md

```diff
@@ -6,7 +6,36 @@ Exercism exercises in Haskell
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
+Haskell is a great language! (no, really!). Here in the Haskell track we do not
+just want to help you learn this great language, but we have the ambition to raise
+and share the passion for it. So we very much appreciate your help to make this happen!
+Any contribution that improves our track is welcome. How could such a contribution
+look like?
+Perhaps you have found a problem in one of the exercises. This can be a simple typo.
+Or is there an essential test case missing? Is the problem description unclear or
+ambiguous? Did you get stuck and would have liked some hints? Or do you question
+the educational value of the exercise? Is it not really Haskell-like?
+Maybe you would love to [port](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track) one of the [missing exercises](http://exercism.io/languages/haskell/todo)? Do you even
+have an idea for a [completely new exercise](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise)?
+Or do you see a potential improvement in something not exercise-related? An example
+could be this text. Or some good book that is missing in the [learning resources](http://exercism.io/languages/haskell/learning).
+
+So you do have an improvement for us? Nice! What's next?
+As a first step we recommend you read the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
+Now ask yourself: What is the nature of your contribution? Is it specific to the
+Haskell track, or is it something that all language tracks could profit from?
+A good example for the latter is an additional test case. Then you should
+contribute to `x-common` as described [here](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#updating-an-exercise-test-suite).
+
+But let's assume your contribution is indeed for the Haskell track. Now you have
+to decide:
+Is it still more in the state of a question or clarification? Maybe you could
```

Comment:
Here too, maybe a bulleted list would make sense and make it harder to miss one of the possible options.